frontend/src/services/validation.js

```diff
@@ -1,4 +1,5 @@
 const emailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const fullNameRegex = /^[a-z]([-']?[a-z]+)*( [a-z]([-']?[a-z]+)*)+$/i;
```

Comment:
why did you add so complicated reg exp for user name? This won't pass cyrillic symbols, and any instead of latin.

For full name only one validation rule should be accepted: not empty